scripts/quests/outlands/An_Undying_Pledge.lua

```diff
@@ -82,19 +80,15 @@ quest.sections =
             ['qm5'] =
             {
                 onTrigger = function(player, npc)
-                    if
-                        quest:getVar(player, 'Prog') == 2 and
-                        player:getCharVar("anUndyingPledgeNM_killed") == 0 and
-                        npcUtil.popFromQM(player, npc, ID.mob.GLYRYVILU, { claim = true, hide = 0 })
-                    then
-                        return player:messageSpecial(ID.text.BODY_NUMB_DREAD)
+                    if quest:getVar(player, 'Prog') == 2 and npcUtil.popFromQM(player, npc, ID.mob.GLYRYVILU, { claim = true, hide = 0 }) then
```

Comment:
multi line this if statement

Reply:
also you deleted the check i[f getcharvar ](player:getCharVar("anUndyingPledgeNM_killed") == 0)
you should convert these variables into quest vars

Reply:
Oh so I don't need to have the mobs lua do this?

Reply:
that makes sense...